src/Arriba/Arriba/ParametersCheckers/ParamChecker.cs

```diff
@@ -53,5 +53,11 @@ public static void ThrowIfNullOrEmpty(this NameValueCollection value, string par
             if (value == null || value.Count == 0)
                 throw new ArgumentException("Not Provided", paramName);
         }
+
+        public static void ThrowIfNull(this IIdentity value, string paramName)
```

Comment:
This should not be required. Actually the param check above in lines 11-22 should be consolidated. The only one that should need to exist is the ThrowIfNull<T>(this T value, string name).

The conditional check should also be: if (value is null)

Reply:
To use the is null for the T I do have to use the version 8.0 for the language definition on Arriba, is that okay?